### PR TITLE
Choice tiles new IA and content updates

### PR DESCRIPTION
**Pull request by chri2547**
image

### CODE REVIEW

File: _component_design/choice-tiles.md [hunk 1]
```diff
@@ -4,42 +4,65 @@ parent: Inputs and Controls
 layout: component
 category: Components
 usage: >
-  Choice tiles are an alternative selector pattern to radio buttons and checkboxes that provide a means of both displaying choices more visually and of providing a more detailed description of a choice.
+  A choice tile represents a selection that a user can make on a page. As an alternative to a radio button or checkbox, a choice tile contains an icon that graphically represents the choice and a description that provides more detail about the choice.
```
Comment (chri2547) [inline _component_design/choice-tiles.md]: choice tile is a selectable object/component. Iconography is optional.

remove stuff about icon. selectable component that you can use a more graphic alternative to a radio and checkbox.

File: _component_design/choice-tiles.md [hunk 2]
```diff
@@ -4,42 +4,65 @@ parent: Inputs and Controls
...
-## Introduction
-
 <div class="hxRow" markdown="1">
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Use choice tiles when you want to use iconography to present a set of choices to the user.
```
Comment (chri2547) [inline _component_design/choice-tiles.md]: Use when you want to increase the selectable space, a larger selectable frame and easier to make a selection.  A choice tile focuses the attention of the user...and allows you to add more context as you can have a name, icon, and description. richer information experience. much bigger click space...richer, in context information. makes choices clearer and its easier to interact with when considering mobile.

File: _component_design/choice-tiles.md [hunk 3]
```diff
@@ -4,42 +4,65 @@ parent: Inputs and Controls
...
 
-- Use choices tiles when presenting a set of choices that are easily recognized by iconography.
-- Only use choice tiles when two or more selections are available.
+## Best practices
 
-### Best practices
+Consider the following best practices when you construct choice tiles:
 
-- When using a description within a choice tile keep the text brief.
+- Only use choice tiles when there are two or more selections available. Do not provide more than nine choice tiles.
```
Comment (chri2547) [inline _component_design/choice-tiles.md]: The number of choice tiles is a function of the number of options, by size of choice tile and the users screen size. Larger sets of choice tiles increases the information processing demands. We recommend more than 1 and fewer than 9.

Comment (chri2547) [reply _component_design/choice-tiles.md]: too many options can lead to option paralysis. Too few is visually confusing. Only as many as you need.

File: _component_design/choice-tiles.md [hunk 4]
```diff
@@ -4,42 +4,65 @@ parent: Inputs and Controls
...
+- Choice tiles do not define specific iconography. Stay within brand guidelines and use iconography that supports the selection contained in a choice tile.
```
Comment (chri2547) [inline _component_design/choice-tiles.md]: no specific set of icons to use with choice tiles, no guidelines, just conform to brand. icon should represent the choice displayed. See the button iconography page for wording, and xref to iconography page.

File: _component_design/choice-tiles.md [hunk 5]
```diff
@@ -4,42 +4,65 @@ parent: Inputs and Controls
...
+- For more complex selections, add an optional description to medium or large choice tiles. Due to space constraints, do not add a description to small choice tiles.
```
Comment (chri2547) [inline _component_design/choice-tiles.md]: Check this against the rest of form inputs.

Comment (chri2547) [reply _component_design/choice-tiles.md]: For more complex information needs...

File: _component_design/choice-tiles.md [hunk 6]
```diff
@@ -4,42 +4,65 @@ parent: Inputs and Controls
...
+- Do not mix sizes in a set of choice tiles.
+- If you do not add a description, collapse the padding to the bottom-most padding.
```
Comment (chri2547) [inline _component_design/choice-tiles.md]: strike this.

File: _component_design/choice-tiles.md [hunk 7]
```diff
@@ -4,42 +4,65 @@ parent: Inputs and Controls
...
+- Limit the description you include on a choice tile to no more than two lines of text.
```
Comment (chri2547) [inline _component_design/choice-tiles.md]: limit description to 100 characters so that the text doesn't flow onto more than 2 lines.

Check character count with Ryan.

File: _component_design/choice-tiles.md [hunk 8]
```diff
@@ -55,12 +78,12 @@ pagelink: https://rackerlabs.github.io/helix-ui/components/choice-tiles/
...
 
-Cards are composed of the following:
+A choice tile contains the following elements:
 
-- **Choice tile**: the container that houses the choice tile selection.
-- **Icon**: an iconographic representation of the selection contained within a choice tile.
-- **Heading**: the title of the selection contained within a choice tile.
-- **Description** *(optional)*: A description briefly explains the choice tile selection to a user. Capitalize the first word in sentences.
+- **Container (required)**: The container holds the icon, heading, and the description.
+- **Icon (required)**: The icon visually represents the choice tile selection.
```
Comment (chri2547) [inline _component_design/choice-tiles.md]: icon is optional.

File: _component_design/choice-tiles.md [hunk 9]
```diff
@@ -55,12 +78,12 @@ pagelink: https://rackerlabs.github.io/helix-ui/components/choice-tiles/
...
+- **Heading (required)**: The heading is the title of the selection contained in the choice tile.
```
Comment (chri2547) [inline _component_design/choice-tiles.md]: Heading = Option label.

File: _component_design/choice-tiles.md [hunk 10]
```diff
@@ -130,23 +149,42 @@ Use the following column widths for the small, medium, and large choice tiles. B
...
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-- Small size choice tiles **cannot** have a description due to the smaller dimensions.
-- An optional description may be added medium and large size choice tiles. Its inclusion is recommended to inform more complex selections.
-- When a description is not used, collapse the padding down to the bottom most padding.
-- Choice tiles do not define specific iconography. Stay within brand guidelines and use iconography that supports the selection contained in a choice tile.
+#### Single-select interactions
+
+A single-select interaction is one where the user .... XXX
```
Comment (chri2547) [inline _component_design/choice-tiles.md]: one is automatically selected by default and selecting one deselects another.

File: _component_design/choice-tiles.md [hunk 11]
```diff
@@ -161,18 +199,20 @@ Use the following column widths for the small, medium, and large choice tiles. B
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Basic states
+#### Multi-select interactions
 
-Choice tiles use most of the same styling that is used for components like radios and checkboxes, with the addition of the checkmark to indicate a selected tile. The user can click on any part of the tile to select it.
+A multi-select interaction is one where the user .... XXX
```
Comment (chri2547) [inline _component_design/choice-tiles.md]: See wording for checkboxes, same principle applies here. Can multi-select.

File: _component_design/choice-tiles.md (hunk 10 above)
Comment (chri2547) [reply _component_design/choice-tiles.md]: saved payment information is common use case.

File: _component_design/choice-tiles.md [hunk 12]
```diff
@@ -183,26 +223,24 @@ For selected states, the iconography changes color to match the given state. In
...
-### Single-select interaction
-
-**Default Interaction**: Like with multi-select, when the user first views the choice tile selector, all tiles are presented in the default state unless a preferred choice is preselected.
+## Layout
 
-**After Selection**: To illustrate that only one selection can be made at a time, the remaining tiles change to the subdued state after a selection is made.
+Use the following guidelines when you position choice tiles on a page.
 
-**Change selection**: If a user wants to change their selection, the subdued state changes to the hover state on hover.
+- Do not span small choice tiles across more than six columns, for a maximum of six tiles in a row.
```
Comment (chri2547) [inline _component_design/choice-tiles.md]: strike 'across more than x columns' out and rephrase.

File: _component_design/choice-tiles.md [hunk 13]
```diff
@@ -217,18 +255,18 @@ For selected states, the iconography changes color to match the given state. In
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Multi-select interaction
+### Responsive specifications
```
Comment (chri2547) [inline _component_design/choice-tiles.md]: Get with Ryan and Roman about this, to see if we need this.

File: _component_design/choice-tiles.md [hunk 14]
```diff
@@ -217,18 +255,18 @@ For selected states, the iconography changes color to match the given state. In
...
```
Comment (chri2547) [inline _component_design/choice-tiles.md]: Check with Ryan about this.

File: _component_design/choice-tiles.md [hunk 15]
```diff
@@ -239,22 +277,46 @@ For selected states, the iconography changes color to match the given state. In
...
 <section class="static-section" markdown="1">
 
-## Handling errors
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+## States
+
+Choice tiles use most of the same styling that is used for components like radios and checkboxes, with the addition of the checkmark to indicate a selected tile. The user can click on any part of the tile to select it.
```
Comment (chri2547) [inline _component_design/choice-tiles.md]: take second sentence and move to best practice.

File: _component_design/choice-tiles.md [hunk 16]
```diff
@@ -239,22 +277,46 @@ For selected states, the iconography changes color to match the given state. In
...
+
+For selected states, the iconography changes color to match the given state. If the color of an icon cannot be changed, be consistent with the color throughout the set of choice tiles. Do not construct some choice tiles to change colors and others that do not change color.
```
Comment (chri2547) [inline _component_design/choice-tiles.md]: Strike 2nd and 3rd sentence.

File: _component_design/choice-tiles.md [hunk 17]
```diff
@@ -239,22 +277,46 @@ For selected states, the iconography changes color to match the given state. In
...
 
 <div class="hxRow" markdown="1">
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Single-select error
+### Required
 
-If there is an error on a selection, change that item to errored state and display an error message below it.
+If a selection is required, add an 4px asterisk to the left of the title an overlap with the gutter. The asterisk inherits the font size of the type style to which it is attached.
```
Comment (chri2547) [inline _component_design/choice-tiles.md]: Look at other inputs for wording.

File: _component_design/choice-tiles.md (hunk 1 above)
Comment (chri2547) [reply _component_design/choice-tiles.md]: Done.

File: _component_design/choice-tiles.md (hunk 2 above)
Comment (chri2547) [reply _component_design/choice-tiles.md]: Done.

File: _component_design/choice-tiles.md (hunk 6 above)
Comment (chri2547) [reply _component_design/choice-tiles.md]: Done.

File: _component_design/choice-tiles.md (hunk 8 above)
Comment (chri2547) [reply _component_design/choice-tiles.md]: Done.

File: _component_design/choice-tiles.md (hunk 9 above)
Comment (chri2547) [reply _component_design/choice-tiles.md]: Done.

File: _component_design/choice-tiles.md (hunk 3 above)
Comment (chri2547) [reply _component_design/choice-tiles.md]: Done.

File: _component_design/choice-tiles.md (hunk 4 above)
Comment (chri2547) [reply _component_design/choice-tiles.md]: Done.

File: _component_design/choice-tiles.md (hunk 5 above)
Comment (chri2547) [reply _component_design/choice-tiles.md]: Done.

File: _component_design/choice-tiles.md (hunk 10 above)
Comment (chri2547) [reply _component_design/choice-tiles.md]: Done.

File: _component_design/choice-tiles.md (hunk 11 above)
Comment (chri2547) [reply _component_design/choice-tiles.md]: Done.

File: _component_design/choice-tiles.md (hunk 12 above)
Comment (chri2547) [reply _component_design/choice-tiles.md]: Done.

File: _component_design/choice-tiles.md (hunk 15 above)
Comment (chri2547) [reply _component_design/choice-tiles.md]: Done.

File: _component_design/choice-tiles.md (hunk 16 above)
Comment (chri2547) [reply _component_design/choice-tiles.md]: Done.

File: _component_design/choice-tiles.md (hunk 17 above)
Comment (chri2547) [reply _component_design/choice-tiles.md]: Done.

File: _component_design/choice-tiles.md [hunk 18]
```diff
@@ -4,42 +4,65 @@ parent: Inputs and Controls
...
 
-### Best practices
+Consider the following best practices when you construct choice tiles:
 
-- When using a description within a choice tile keep the text brief.
+- Only use as many choice tiles as you need. Too few choice tiles (for example, 2) can be visually confusing to the user while larger sets of choice tiles increase the user's information processing demands and can lead to *option paralysis*. The number of choice tiles is also a function of the choice tile size and the user's screen size. We recommend using choice tiles when there is more than one and a maximum of nine choices.
+- Construct a choice tile so that the user can click on any part of the tile to select it.
+- You can design your own icon as long as it conforms to the Rackspace brand. If there is an established meaning to an icon, don’t use it to mean something else. See [iconography]({{site.baseurl}}/style/iconography.html) for more information about icons.
+- For more complex information needs, add a description to medium or large choice tiles. Due to space constraints, do not add a description to small choice tiles.
+- Do not mix sizes in a set of choice tiles.
```
Comment (chri2547) [inline _component_design/choice-tiles.md]: Ensure that choice tiles are the same size.

File: _component_design/choice-tiles.md [hunk 19]
```diff
@@ -183,26 +221,52 @@ For selected states, the iconography changes color to match the given state. In
 
 <section class="static-section" markdown="1">
 
-## Interactions
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+## Layout
```
Comment (chri2547) [inline _component_design/choice-tiles.md]: REmove this section.

File: _component_design/choice-tiles.md [hunk 20]
```diff
@@ -183,26 +221,52 @@ For selected states, the iconography changes color to match the given state. In
...
+</section>
+
+<section class="static-section" markdown="1">
 
 <div class="hxRow" markdown="1">
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Single-select interaction
+### Responsive specifications
```
Comment (chri2547) [inline _component_design/choice-tiles.md]: Remove this section.

File: _component_design/choice-tiles.md (hunk 14 above)
Comment (chri2547) [reply _component_design/choice-tiles.md]: NA.

File: _component_design/choice-tiles.md (hunk 20 above)
Comment (chri2547) [reply _component_design/choice-tiles.md]: Done.

File: _component_design/choice-tiles.md (hunk 19 above)
Comment (chri2547) [reply _component_design/choice-tiles.md]: Done.

File: _component_design/choice-tiles.md (hunk 18 above)
Comment (chri2547) [reply _component_design/choice-tiles.md]: Done.

File: _component_design/choice-tiles.md [hunk 21]
```diff
@@ -55,19 +79,19 @@ pagelink: https://rackerlabs.github.io/helix-ui/components/choice-tiles/
...
-Cards are composed of the following:
+A choice tile contains the following elements:
 
-- **Choice tile**: the container that houses the choice tile selection.
-- **Icon**: an iconographic representation of the selection contained within a choice tile.
-- **Heading**: the title of the selection contained within a choice tile.
-- **Description** *(optional)*: A description briefly explains the choice tile selection to a user. Capitalize the first word in sentences.
+- **Container (required)**: The container holds the icon, heading, and the description.
+- **Icon (optional)**: The icon visually represents the choice tile selection.
+- **Option label (required)**: The option label is the title of the selection contained in the choice tile.
```
Comment (palindromebart) [inline _component_design/choice-tiles.md]: is the title of the option you can select with the choice tile.

File: _component_design/choice-tiles.md [hunk 22]
```diff
@@ -55,19 +79,19 @@ pagelink: https://rackerlabs.github.io/helix-ui/components/choice-tiles/
...
+- **Description (optional)**: A description briefly explains the choice tile selection to a user.
```
Comment (palindromebart) [inline _component_design/choice-tiles.md]: explains the option or the impact of selecting the option you can select with the choice tile.

File: _component_design/choice-tiles.md [hunk 23]
```diff
@@ -161,18 +170,22 @@ Use the following column widths for the small, medium, and large choice tiles. B
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Basic states
+#### Single-select interactions
+
+A single-select interaction is analogous to radios where the user can only select one choice tile at a time.
+
+In the default state, the user can click any of the choice tiles. You can construct choices tiles so that one of the options is selected by default. The user can then make another selection, if necessary. When the user selects an option, the previously selected choice tile becomes deselected and all remaining tiles change to the subdued state. The subdued state reinforces that only one choice tile can be selected.
```
Comment (palindromebart) [inline _component_design/choice-tiles.md]: this is a really long chunk of text. can you break it up?

Comment (palindromebart) [inline _component_design/choice-tiles.md]: Also, we should definitely add a line about the possibility of  having one tile selected by default. you can look to radios for the verbiage we used there.

File: _component_design/choice-tiles.md [hunk 24]
```diff
@@ -183,26 +196,22 @@ For selected states, the iconography changes color to match the given state. In
...
-### Single-select interaction
-
-**Default Interaction**: Like with multi-select, when the user first views the choice tile selector, all tiles are presented in the default state unless a preferred choice is preselected.
+#### Multi-select interactions
 
-**After Selection**: To illustrate that only one selection can be made at a time, the remaining tiles change to the subdued state after a selection is made.
+A multi-select interaction is analogous to checkboxes where the user can select multiple choice tiles at a time.
 
-**Change selection**: If a user wants to change their selection, the subdued state changes to the hover state on hover.
+In the default state, the user can click any of the choice tiles. You can construct choice tiles so that one of the options is selected by default. The user can then make another selection, if necessary. When the user selects an option, the unselected choice tiles remain in the default state. The default state reinforces that multiple choice tiles can be selected.
```
Comment (palindromebart) [inline _component_design/choice-tiles.md]: can we break this up?

File: _component_design/choice-tiles.md [hunk 25]
```diff
@@ -215,20 +224,36 @@ For selected states, the iconography changes color to match the given state. In
...
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Multi-select interaction
+### Enabled
 
-**Default interaction**: When the user first views the choice tile selector, all tiles are presented in the default state unless a preferred choice is preselected.
+Choice tiles use most of the same styling that is used for components like radios and checkboxes, with the addition of the checkmark to indicate a selected tile.
 
-**After selection**: After a selection is made, the selected tile changes to the selected state and the other tiles remain in the default state.
+For selected states, the iconography changes color to match the given state. If you cannot adjust the color of an icon, be consistent with your color decision throughout the choice tile set. Do not have some icons that change color and others that do not change color. When possible, change the icon color with the selected states.
```
Comment (palindromebart) [inline _component_design/choice-tiles.md]: maybe move the last sentence to the top? it's redundant currently, and i kinda like the last sentence phrasing better.

File: _component_design/choice-tiles.md [hunk 26]
```diff
@@ -215,20 +224,36 @@ For selected states, the iconography changes color to match the given state. In
...
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Multi-select interaction
+### Enabled
```
Comment (palindromebart) [inline _component_design/choice-tiles.md]: i don't think this visual is accurate (see the opacity changes...)

File: _component_design/choice-tiles.md [hunk 27]
```diff
@@ -239,22 +264,22 @@ For selected states, the iconography changes color to match the given state. In
...
 <section class="static-section" markdown="1">
 
-## Handling errors
-
 <div class="hxRow" markdown="1">
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Single-select error
+### Disabled
```
Comment (palindromebart) [inline _component_design/choice-tiles.md]: the visual is definitely not the right one here.

File: _component_design/choice-tiles.md [hunk 28]
```diff
@@ -239,22 +264,22 @@ For selected states, the iconography changes color to match the given state. In
...
+
+When possible, inform the user about why an option is not available. Consider adding a tooltip to a disabled choice tile explaining why it is disabled and how to enable it.
 
-If there is an error on a selection, change that item to errored state and display an error message below it.
+When you cannot control the color of the icon, set the icon opacity to 30%.
```
Comment (palindromebart) [inline _component_design/choice-tiles.md]: did this get approved by Ryan? we should check on this.

File: _component_design/choice-tiles.md (hunk 26 above)
Comment (palindromebart) [reply _component_design/choice-tiles.md]: also there are a few tweaks needed to the visual. e.g. 
- icon and option label aren't totally centered in 
the tile. 
- some say "Option Label" some say "Label"  let's standardize on "Option Label"

File: _component_design/choice-tiles.md [hunk 29]
```diff
@@ -269,16 +294,16 @@ If there is an error on a selection, change that item to errored state and displ
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Group-select error
+### Single-select error
```
Comment (palindromebart) [inline _component_design/choice-tiles.md]: there are a few tweaks needed to the visual here too. e.g. 
- icon and option label aren't totally centered in 
the tile. 
- some say "Option Label" some say "Label"  let's standardize on "Option Label"

File: _component_design/choice-tiles.md [hunk 30]
```diff
@@ -4,42 +4,66 @@ parent: Inputs and Controls
...
-## Introduction
-
 <div class="hxRow" markdown="1">
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Like radios and checkboxes, choice tiles provide the user a means of making a decision and selecting from a group of options. Because choice tiles increase the selectable space of a choice, you can add an icon and description, which provides context and a richer information experience to the user. Choice tiles help decision making by focusing the user's attention.
```
[the remark this answers is not in the document]
Comment (chri2547) [reply _component_design/choice-tiles.md]: Done.

File: _component_design/choice-tiles.md [hunk 31]
```diff
@@ -4,42 +4,66 @@ parent: Inputs and Controls
...
-- Only use choice tiles when two or more selections are available.
+## Best practices
 
-### Best practices
+Consider the following best practices when you construct choice tiles:
 
-- When using a description within a choice tile keep the text brief.
+- Only use as many choice tiles as you need. Too few choice tiles (for example, 2) can be visually confusing to the user while larger sets of choice tiles increase the user's information processing demands and can lead to *option paralysis*. The number of choice tiles is also a function of the choice tile size and the user's screen size. We recommend using choice tiles when there is more than one and a maximum of nine choices.
+- Construct a choice tile so that the user can click on any part of the tile to select it.
+- You can design your own icon as long as it conforms to the Rackspace brand. If there is an established meaning to an icon, don’t use it to mean something else. See [iconography]({{site.baseurl}}/style/iconography.html) for more information about icons.
```
[the remark this answers is not in the document]
Comment (chri2547) [reply _component_design/choice-tiles.md]: Done.

File: _component_design/choice-tiles.md [hunk 32]
```diff
@@ -4,42 +4,66 @@ parent: Inputs and Controls
...
+- For more complex information needs, add a description to medium or large choice tiles. Due to space constraints, do not add a description to small choice tiles.
+- Do not mix small, medium, and large size choice tiles in a choice tile set.
+- Within a choice tile set, ensure that choice tiles are all uniform in size.
```
[the remark this answers is not in the document]
Comment (chri2547) [reply _component_design/choice-tiles.md]: Done.

File: _component_design/choice-tiles.md (hunk 21 above)
Comment (chri2547) [reply _component_design/choice-tiles.md]: Done.

File: _component_design/choice-tiles.md (hunk 25 above)
Comment (chri2547) [reply _component_design/choice-tiles.md]: Done.

File: _component_design/choice-tiles.md (hunk 27 above)
Comment (chri2547) [reply _component_design/choice-tiles.md]: That is totally by bad - fixed!

**Comment by palindromebart** [comment]
Need to change the name of this component to Option Tile, per bowling decision on May 15th 2018. this way we're aligned with HelixUI naming

File: _component_design/choice-tiles.md (hunk 29 above)
Comment (chri2547) [reply _component_design/choice-tiles.md]: roman updated the visuals, done.

File: _component_design/choice-tiles.md (hunk 28 above)
Comment (chri2547) [reply _component_design/choice-tiles.md]: on 1/29 we will discuss in bowling.

File: _component_design/choice-tiles.md (hunk 26 above)
Comment (chri2547) [reply _component_design/choice-tiles.md]: Comment 1: will discuss in bowling on 1/29
Comment 2: Done, roman updated visuals.

**Comment by cyrichardson** [comment]
@chri2547 On it.

**Comment by cyrichardson** [comment]
@chri2547 Suggested changes here: 
[helix_choice-tiles_cyrEdits.docx](https://github.com/rackerlabs/design-system/files/2809293/helix_choice-tiles_cyrEdits.docx)

File: _component_design/choice-tiles.md (hunk 26 above)
Comment (chri2547) [reply _component_design/choice-tiles.md]: Removed reference to opacity, done.

**Comment by cyrichardson** [comment]
@chri2547 Sorry I picked up the wrong one. On one suggested for options tiles here: 
[helix_option-tiles_cyrEdits.docx](https://github.com/rackerlabs/design-system/files/2809643/helix_option-tiles_cyrEdits.docx)

File: _component_design/option-tiles.md [hunk 33]
```diff
@@ -0,0 +1,338 @@
...
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Like radios and checkboxes, option tiles provide the user a means of selecting from a group of options. Because option tiles increase the selectable space of a choice, you can add an icon and description, which provides context and a richer information experience to the user. Option tiles help decision making by focusing the user's attention.
```
Comment (palindromebart) [inline _component_design/option-tiles.md]: invert this first sentence so it talks about option tiles first and then related them to radios and checkboxes.

File: _component_design/option-tiles.md [hunk 34]
```diff
@@ -0,0 +1,338 @@
...
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Best practices
+
+Consider the following best practices when you construct option tiles:
+
+- Only use as many option tiles as you need. Too few option tiles (for example, two option tiles) can be visually confusing to the user while larger sets of option tiles increase the user's information processing demands and can lead to *option paralysis*. The number of option tiles is also a function of the option tile size and the user's screen size. We recommend using option tiles when there is more than one and a maximum of nine choices.
```
Comment (palindromebart) [inline _component_design/option-tiles.md]: 1) this feels like a LOT of text. can we shorten it? 
Only use as many option tiles as you need. We recommend between 2 and 9 to avoid visual confusion or option paralysis.

File: _component_design/option-tiles.md [hunk 35]
```diff
@@ -0,0 +1,338 @@
...
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+#### Single-select interactions
+
+A single-select interaction is analogous to radios where the user can only select one option tile at a time.
+
+In the default state, the user can click any of the option tiles. You can construct option tiles so that one of the options is selected by default. The user can then make another selection, if necessary. When the user selects an option, the previously selected option tile becomes deselected and all remaining tiles change to the subdued state. The subdued state reinforces that only one option tile can be selected.
```
Comment (palindromebart) [inline _component_design/option-tiles.md]: Should we break this up into bullets? it feels hard to parse and totally liable to be skipped.

File: _component_design/option-tiles.md [hunk 36]
```diff
@@ -0,0 +1,338 @@
...
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+#### Multi-select interactions
+
+A multi-select interaction is analogous to checkboxes where the user can select multiple option tiles at a time.
+
+In the default state, the user can click any of the option tiles. You can construct option tiles so that one of the options is selected by default. The user can then make another selection, if necessary. When the user selects an option, the unselected option tiles remain in the default state. The default state reinforces that multiple option tiles can be selected.
```
Comment (palindromebart) [inline _component_design/option-tiles.md]: Same here for the bullets/skipping comment above

File: _component_design/option-tiles.md (hunk 33 above)
Comment (chri2547) [reply _component_design/option-tiles.md]: Done.

File: _component_design/option-tiles.md (hunk 34 above)
Comment (chri2547) [reply _component_design/option-tiles.md]: Done.

File: _component_design/option-tiles.md (hunk 35 above)
Comment (chri2547) [reply _component_design/option-tiles.md]: I shortened this text quite a bit, by about 1/2 I think. See how that works for you now. I agree, it was a bit of a mouthful to get through! :)

File: _component_design/option-tiles.md (hunk 36 above)
Comment (chri2547) [reply _component_design/option-tiles.md]: Also shortened this text quite a bit. See how it works for ya!

**Comment by palindromebart** [comment]
Ship it, Squirrel!

File: _docs/style/focus-states.md [hunk 37]
```diff
@@ -364,7 +364,7 @@ The beacon comes with a glow baked in, so when it receives focus, the existing g
 
 ### Focusable containers
 
-Some elements, such as [Choice Tiles]() feature a clickable container. In these cases, focus is applied to the entire container, according to normal focus rules.
+Some elements, such as [Option Tiles]() feature a clickable container. In these cases, focus is applied to the entire container, according to normal focus rules.
```
Comment (nimbinatus) [inline _docs/style/focus-states.md]:
```suggestion
Some elements, such as [Option Tiles]({{site.baseurl}}/components/option-tiles.html) feature a clickable container. In these cases, focus is applied to the entire container, according to normal focus rules.
```

File: _component_design/option-tiles.md [hunk 38]
```diff
@@ -0,0 +1,338 @@
...
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Option tiles, like radios and checkboxes, provide the user a means of selecting from a group of options. Because option tiles increase the selectable space of a choice, you can add an icon and description, which provides context and a richer information experience to the user. Option tiles help decision making by focusing the user's attention.
```
[the remark this answers is not in the document]
Comment (chri2547) [reply _component_design/option-tiles.md]: Done.

File: _component_design/option-tiles.md [hunk 39]
```diff
@@ -0,0 +1,338 @@
...
+layout: component
+category: Components
+usage: >
+  An option tile is a selectable component that graphically represents a choice a user can make on a page.
+preview-image: preview-images/option-tile.svg
+status: stable
+resource: true
+last-modified: 2019-02-06
+helix-ui-css: false
+helix-ui-javascript: false
```
[the remark this answers is not in the document]
Comment (chri2547) [reply _component_design/option-tiles.md]: Done.

File: _docs/style/focus-states.md (hunk 37 above)
Comment (chri2547) [reply _docs/style/focus-states.md]: The focusable containers section is no longer available. I'm not sure why you saw this?
